BBCIngest/MainForm.cs

```diff
@@ -118,8 +118,12 @@ private void buttonRemoveTasks_Click(object sender, EventArgs e)
 
         private void createTask(ScheduleInstaller schedule)
         {
-            string path = System.Diagnostics.Process.GetCurrentProcess().MainModule.FileName;
-            TaskDefinition td = schedule.createTaskDefinition(path);
+            string progPath = System.Diagnostics.Process.GetCurrentProcess().MainModule.FileName;
+            string arguments = "once";
```

Comment:
Just checking that we always want to send the argument "once" irrespective of having a custom config path or just the default path?